### DISCUSSION TITLE
Close on gesture up doesn't work

### DISCUSSION THREAD

**Discussion by jimdigrease** [igordanchenko/yet-another-react-lightbox#232] [Q&A] [answered]
With this settings 

```js
controller={{
  closeOnPullUp: true,
  closeOnPullDown: true,
  closeOnBackdropClick: true
}}
```

all work as expected except for closeOnPullUp (in Chrome and Opera browsers)

**Comment by igordanchenko** [comment, marked as answer]
Have you updated to the latest version? `closeOnPullUp` option was released in [v3.16.0](https://github.com/igordanchenko/yet-another-react-lightbox/releases/tag/v3.16.0)

**Reply by jimdigrease** [reply]
Thanks for help, just updated and it started do work)
